.circleci/config.yml

```diff
@@ -83,6 +83,28 @@ jobs:
       - run:
```

Comment:
So pushed to Docker Hub regardless of CI. Though it was this way before too. Ok.